lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -3480,8 +3480,27 @@ LogicalResult FIRRTLLowering::visitExpr(MultibitMuxOp op) {
       return failure();
     loweredInputs.push_back(lowered);
   }
+
+  // We lower multbit mux into array indexing with vendor pragmas in the
+  // following form.
+  //
+  // wire GEN;
+  // (* synopsys infer_mux_override *)
+  // assign GEN = array[index] (* cadence map_to_mux *);
```

Comment:
I think this comment is a little off from the actual form now.

Reply:
Nice catch, thanks!